ACC2024/RobotDynamics/@RigidBody2D/RigidBody2D.m

```diff
@@ -46,9 +46,15 @@
             end
 
         end
-
+        
+        % [x,y,theta]
         function position = forwardKinematics(obj, pointInBodyFrame)
             position = obj.CoordinateFrame.HomogeneousFromBase * [pointInBodyFrame.ColumnVector; 1];
+            position = [position; obj.CoordinateFrame.RotationFromBase.Theta];
```

Comment:
If you want to return [x,y,theta], then line 53 is not needed. Line 52 is a column vector with [x,y,theta].